geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa.properties

```diff
@@ -1,4 +1,39 @@
 geomesa.project.version=${project.version}
 geomesa.build.date=${git.build.time}
 geomesa.build.commit.id=${git.commit.id}
-geomesa.build.branch=${git.branch}
\ No newline at end of file
+geomesa.build.branch=${git.branch}
+
+#- QueryProperties
+geomesa.force.count="false"
```

Comment:
is there a way we can keep these + the ones in the default site.xml in a single place, so that we don't have to modify it two places? Possibly put `geomesa-site.xml.template` in src/main/resources and load it from there, plus copy it into tools for the distribution.